Cargo.toml

```diff
@@ -10,7 +10,7 @@ homepage = "https://github.com/dusk-network/bls12_381"
 license = "MIT/Apache-2.0/MPL-2.0"
 name = "dusk-bls12_381"
 repository = "https://github.com/dusk-network/bls12_381"
-version = "0.14.1"
+version = "0.14.2"
```

Comment:
This will also be changed in the PR that releases a new version.

Reply:
Reverted